src/test/java/org/checkerframework/specimin/SyntheticAnnotationsTest.java

```diff
@@ -0,0 +1,18 @@
+package org.checkerframework.specimin;
+
+import java.io.IOException;
+import org.junit.Test;
+
+/**
+ * This test checks that if Specimin will work properly where there is a super constructor while the
+ * parent class file is not in the root directory physically
```

Comment:
This comment doesn't seem correct (maybe copied from a different test?), and it should be updated.